pkg/esp32_sdk/patches/0006-compilation-include-missing-header-files.patch

```diff
@@ -47,16 +47,16 @@ index f5a6b292..6f9fd2b1 100644
  #include "soc/soc_caps.h"
  #include "regi2c_ctrl.h"
 diff --git a/components/nvs_flash/src/nvs_encrypted_partition.cpp b/components/nvs_flash/src/nvs_encrypted_partition.cpp
-index 26e8a331..4de077b9 100644
+index 653fbefa..d614fa61 100644
 --- a/components/nvs_flash/src/nvs_encrypted_partition.cpp
 +++ b/components/nvs_flash/src/nvs_encrypted_partition.cpp
-@@ -13,6 +13,7 @@
- // limitations under the License.
- 
+@@ -7,6 +7,7 @@
  #include <cstring>
-+#include "nvs.hpp"
  #include "nvs_encrypted_partition.hpp"
  #include "nvs_types.hpp"
++#include "nvs.hpp"
+ 
+ namespace nvs {
```

Comment:
Shouldn't there also be a closing curly bracket?

Reply:
The namespace is in the file being patched, not the patch